### ISSUE TITLE
Alexa Media shows as "Not Loaded"

### ISSUE THREAD

**Issue by otomay** [custom-components/alexa_media_player#1469]
**Describe the bug**
The integration doesn't load. All entities are showing as unavaiable. It was working perfectly until 2/3 days for 6 months +.

**To Reproduce**
1. Just start/restart the HA server and there we go.

**Expected behavior**
To work.

**Screenshots**

**System details**
 - Home-assistant (version): Home Assistant 2021.12.3
 - Hassio (Yes/No): No
 - alexa_media  (version from `const.py` or HA startup): 3.10.15
 - alexapy (version from `pip show alexapy` or HA startup): Can't find it 
 - Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). We will not debug login issues if unanswered: y

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component). We'll be most likely asking for them anyway.

[homeassistant.log](https://github.com/custom-components/alexa_media_player/files/7741607/homeassistant.log)

**Additional context**
I can't configure the advanced debug. It's says the config file has smt wrong.

Also I can't delete the integration to try a fresh start, it just stay there. I can remove the custom component in HACS section, but the account stay in integrations page. If I hit the exclude button and confirm it nothing happens. When I try to add a new alexa_media integration it says "It is not possible to load the configuration flow".

Nothing different was installed to trigger this, just maybe the updates from HA, hacs, etc.

**Comment by coolhansen** [comment]
It's the core-update to 2021.12.3.
I restored my backup and all is ok again.

**Comment by otomay** [comment]
You're right. I tried to restore 2021.12.2, but the issue persisted, so I restored the 2021.11.5 and it did the trick. Also has improved performance for tuya v2 integration (started to show some connectivity problems too after the update). Thank you!

**Comment by otomay** [comment]
Still with the problem with the 12.7 update... has anyone experienced that too?

**Comment by luigixiv** [comment]
Hi.
Me too, I have the same issue with the version 12.6 of the core.

**Comment by luigixiv** [comment]
The issue has been magically solved by itself.
I just restarted HA and now alexa_media is working again.

I had the same issue with notification from Telegram this morning, because of the notify service did not load properly.
So maybe the issues are related: you cannot send notification to alexa because of the notify service did not load.

**Comment by coolhansen** [comment]
I solved it this way:
- update HA Core and reboot
- update AM
- force reboot via disconnect/reconnect the power

**Comment by alandtse** [comment]
Your log is truncated so isn't helpful. However, please ensure you're using 3.10.15 with the latest version of HA. The logs indicate you're on an older version despite your statement in the template.

**Comment by otomay** [comment]
It was all updated. Just worked poorly in 11.5 with tuya v2 laggin'.  I tried everything, no success... so I backed-up the txt from lovelace and nodered and did a fresh start on the server. Now it's working perfectly. It's a bit sad, cause I lost some hours just to be exactly where I was in the start, but at least I'm good now. The instalation was a bit older, 1y+, and I noticed that some instalation procedures happend in a different way this time (server, integrations). Maybe it was some problem during the updates because of this news procedures, idk... Should I close the issue?
